### PR TITLE
Attempt 3

### PR DESCRIPTION
**Pull request by jonathankwok**
### What this does:

Aligns mis-aligned images and text content. 
### Before:

image
### After:

image
### Added bonus:

Corrected some trailing white space issues.

For review: @jules2689 @karanthukral @lauriermantel
FYI: @justinthec

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -57,7 +57,7 @@ p {
 }
 
 .connect-icon path:hover {
-  fill: #EF5350; 
+  fill: #EF5350;
```
Comment (jules2689) [inline css/style.css]: I know this wasn't you, but it is 2016. We probably shouldn't be using CSS. Can you take a look at integrating a proper stylesheet system like SASS?

File: index.html [hunk 2]
```diff
@@ -14,7 +14,7 @@
         <!-- ============ Materialize core CSS ============ -->
```
Comment (jules2689) [inline index.html]: This also wasn't you, but these comments are superfluous and not very useful. We can see that it is importing materialize's css, we don't need 2 lines telling us the same thing.

File: index.html [hunk 3]
```diff
@@ -122,7 +122,7 @@ <h3 class="center">Our Sponsors</h3>
           <!--   Icon Section   -->
           <div class="row">
             <div class="col s12 m4 valign">
-                <h2 class="center"><img class="spsr-logo" src="http://www.kw4rent.com/wp-content/themes/twentyeleven/images/logo.png"></h2>
+                <div class="center"><img class="spsr-logo" src="http://www.kw4rent.com/wp-content/themes/twentyeleven/images/logo.png"></div>
```
Comment (jules2689) [inline index.html]: :+1: Images shouldn't be in header tags. Good job!

**Comment by karanthukral** [comment]
LGTM.

**Comment by jules2689** [comment]
One thing we should be aware of is that IE and Edge do not support `object-fit`, but all *real* browsers do. Considering this is a webpage for a Hong Kong association and HK has about [15% usage of IE and Edge](https://clicky.com/marketshare/hk/web-browsers/), we may want to look into a polyfill to accommodate those users. While I know the association itself would not be from HK, it does make sense that some family and friends from HK may be viewing the site.

That being said, I don't think not having the polyfill would greatly degrade the experience for those users as the experience would be akin to what is currently available from @justinthec, so the polyfill dependency may not be worth it.

LGTM other than the minor comments and concerns about code outside this PR. 
:+1: good catch @jonathankwok

**Comment by LaurierMantel** [comment]
Code looks good to me  @justinthec @jonathankwok @jules2689 @karanthukral, has anyone tested these changes locally?

**Comment by jules2689** [comment]
Yea, forgot to mention that. 
:tophat:'d the changes.

**Comment by justinthec** [comment]
You all need to off yourselves

**Comment by jonathankwok** [comment]
So, what's the final status on this? Can we merge?

**Comment by LaurierMantel** [comment]
:+1: LGTM, :ship: away

**Comment by LaurierMantel** [comment]
I think we should go back to #6, especially Julian's comment.  We wouldn't have this kind of problems if we were all made collaborators

**Comment by karanthukral** [comment]
http://motionlmags.tumblr.com/post/60938992929/love-emiliacorona204-hotmail-com-started-following-you

**Comment by karanthukral** [comment]
giphy

**Comment by LaurierMantel** [comment]
YAAAAS

**Pull request merged**